bottles/backend/managers/manager.py

```diff
@@ -1015,6 +1017,25 @@ def update_config(
 
         return Result(status=True, data={"config": config})
 
+    @staticmethod
+    def __chattr_f(directory: str):
```

Comment:
It would be better to move this to the [file utils](https://github.com/bottlesdevs/Bottles/blob/main/bottles/backend/utils/file.py)

Reply:
Sure thing, done :ok_hand: